docs/design/configuration/architecture_file.md

```diff
@@ -100,3 +106,22 @@ nodes:
         subscription_topic_name: /pong
         publisher_topic_name: /ping
 ```
+
+## Callback identification
+
+`callback_name` is a name used by users to identify a callback.
+
+CARET reads trace data and loads callback information.
+Callback addresses can be obtained from trace data,
+but callback_name cannot be obtained because ROS does not have the ability to assign callback_names.
+
+After loading information, CARET binds the `callback_name` and callback address by following items.
+
+- node_name
+- callback_type
+- period_ns / topic_name
+- symbol
+- construction_order
+
+By using this information to match `callback_name` and callback address,
+`callback_name` will always refer to identical callbacks, regardless of the number of recordings.
```

Comment:
```suggestion
each `callback_name` will always be unique if there are multiple callbacks instantiated from a same function definition.
```

Reply:
I added this note because there was a part that was difficult to understand.
Please confirm.

d25ef82